shell/server/pre-meteor.js

```diff
@@ -43,6 +43,12 @@ function isSandstormShell(hostname) {
   return (hostname === HOSTNAME || (DDP_HOSTNAME && hostname === DDP_HOSTNAME));
 }
 
+function hasPseudoTLD(hostname) {
+  // Returns true if this hostname ends in one of the major pseudo-TLDs.
+  let tld = hostname.split(".").pop();
+  return (tld === ("onion" || "i2p" || "gnu"));
```

Comment:
Note that elsewhere in the codebase, we seem to use `_.contains(array, member)`, based on this grep:

```
$ git grep _.contains
shell/packages/sandstorm-accounts-oauth/oauth_server.js:    if (!_.contains(Accounts.oauth.serviceNames(), result.serviceName)) {
shell/packages/sandstorm-accounts-packages/accounts.js:  if (_.contains(services, serviceName)) {
shell/packages/sandstorm-accounts-packages/accounts.js:  if (!_.contains(services, serviceName)) {
shell/packages/sandstorm-accounts-ui/accounts_ui.js:    if (!_.contains(VALID_KEYS, key))
shell/packages/sandstorm-accounts-ui/accounts_ui.js:    if (_.contains([
shell/packages/sandstorm-accounts-ui/login_buttons_dropdown.js:    return _.contains(
shell/packages/sandstorm-accounts-ui/login_buttons_session.js:  if (!_.contains(VALID_KEYS, key))
shell/packages/sandstorm-accounts-ui/login_buttons_session.js:    if (_.contains(['errorMessage', 'infoMessage'], key))
shell/packages/sandstorm-accounts-ui/login_buttons_single.js:  if (_.contains(_.pluck(getLoginServices(), "name"), attemptInfo.type))
shell/server/proxy.js:          !(_.contains(["GET", "HEAD", "POST", "PUT", "DELETE"], requestedMethod))) {
```